Source/NavierStokes.cpp

```diff
@@ -480,16 +472,16 @@ NavierStokes::predict_velocity (Real  dt)
 
     // Floor small values of states to be extrapolated
 #ifdef _OPENMP
-#pragma omp parallel
+#pragma omp parallel if (Gpu::notInLaunchRegion())
 #endif
     for (MFIter mfi(Umf,TilingIfNotGPU()); mfi.isValid(); ++mfi)
     {
         Box gbx=mfi.growntilebox(Godunov::hypgrow());
         auto const& fab_a = Umf.array(mfi);
-        AMREX_HOST_DEVICE_FOR_4D ( gbx, BL_SPACEDIM, i, j, k, n,
+        AMREX_PARALLEL_FOR_4D ( gbx, BL_SPACEDIM, i, j, k, n,
         {
             auto& val = fab_a(i,j,k,n);
-            val = amrex::Math::abs(val) > 1.e-20 ? val : 0;
+            val = std::abs(val) > 1.e-20 ? val : 0;
```

Comment:
I'm not sure, but I think this has to be ``amex::Math:abs`` to run on Summit and other upcoming systems.

Reply:
I was just trying to make the flooring loop consistent (as it appears twice in the code). Looks like I choose the wrong option. Making a function is a better way to ensure consistency anyway.